src/resolvers-cassandra/Settings/mutations.js

```diff
@@ -19,13 +20,64 @@ const TRUSTED_SOURCES_RANK_DEFAULT = 10;
 function createOrReplaceSite(args, res) { // eslint-disable-line no-unused-vars
 }
 
+function facebookPagePrimaryKeyValuesToRowKey(values) {
+  return [ TRUSTED_SOURCES_CONNECTOR_FACEBOOK, values[1], values[2] ];
+}
+
+function facebookPageRowKeyToPrimaryKey(page) {
+  const params = page.RowKey.split(',');
+  if (params.length != 3) {
+    throw('Expecting three element comma-delimited RowKey representing (connector, sourceid, sourcetype).');
+  }
+  return facebookPagePrimaryKeyValuesToRowKey(params);
+}
+
+function normalizedFacebookPage(primaryKeyValues) {
+  return {
+    RowKey: facebookPagePrimaryKeyValuesToRowKey(primaryKeyValues),
+    pageUrl: primaryKeyValues[1]
+  };
+}
+
 /**
  * @param {{input: {site: string, pages: Array<{pageUrl: string, RowKey: string}>}}} args
  * @returns {Promise.<{runTime: string, pages: Array<{pageUrl: string, RowKey: string}>}>}
  */
 function modifyFacebookPages(args, res) { // eslint-disable-line no-unused-vars
   return new Promise((resolve, reject) => {
-    reject('Modification not yet supported.');
+    const pages = args && args.input && args.input.pages;
+    if (!pages || !pages.length) return reject('No pages specified');
+    
+    const invalidPages = pages.filter(pages=>!pages.pageUrl);
```

Comment:
FYI: Unless you want to provide the nice error message, you can also use `pages.some(page => !page.pageUrl)`